Wabbajack.App.Wpf/Models/LogStream.cs

```diff
@@ -59,6 +59,8 @@ protected override void Dispose(bool disposing)
 
     protected override void Write(LogEventInfo logEvent)
     {
+        if(logEvent.Level < LogLevel.Error)
+            return;
```

Comment:
The log is referred to a lot when checking what's going on, not sure about this. Maybe better suited for a separate PR to discuss this.

Reply:
Yeah, this aught to be out of scope. Will remove and suggest separately.